src/tooling/simulating/index.md

```diff
@@ -4,7 +4,7 @@ Simulating projects can be very handy. It allows users to test projects using CI
 
 At the moment, there are a few ways of simulating Rust projects on Espressif chips, all of them have some limitations, but it's quickly evolving and getting better every day.
 
-In this chapter, we will discuss the different simulation tools that are available.
+In this chapter, we will discuss the different available simulation tools.
```

Comment:
I think we can omit "different", because nothing in this sentence implies that the tools are very similar.

However, it might be good to emphasize that the tools we are about to go through are available at this time. Because new tools might appear in the future.

How about:

... we will discuss currently available simulation tools.